plugins/k8smeta/src/plugin.cpp

```diff
@@ -138,8 +138,8 @@ falcosecurity::init_schema my_plugin::get_init_schema()
 		},
 		"hostProc": {
 			"type": "string",
-			"title": "Path to reach the '/proc' folder we want to scan.",
-			"description": "The plugin needs to scan the '/proc' of the host on which is running. In Falco usually we put the host '/proc' folder under '/host/proc' so the the default for this config is '/host'. The path used here must not have a final '/'."
+			"title": "[DEPRECATED] Path to reach the '/proc' folder we want to scan.",
```

Comment:
Deprecated initConfig key, unused now.

Reply:
Do you confirm that this setting won't produce any effect anymore?

If so, we should add a notice in addition to the deprecation. Make sense?

Reply:
Done